src/Plate/index.tsx

```diff
@@ -130,10 +130,10 @@ export function positionForCoordinates(
 
 export function wellAtPosition(
   position: number,
-  data: Array<PlateWell>,
+  data: Array<PlateWell> | null,
```

Comment:
```suggestion
  data: Array<PlateWell>,
```

It is already obvious this function will return`null` if `data == null`, the caller should check.